functions/Install-DbaFirstResponderKit.ps1

```diff
@@ -11,34 +11,24 @@ First Responder Kit links:
 http://FirstResponderKit.org
 https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit
 
-
 .PARAMETER SqlInstance
 SQL Server instance or collection of SQL Server instances
 
 .PARAMETER Database
-Database to store the FRK stored procs, typically master
+Database to store the FRK stored procs, typically master and master by default
 
 .PARAMETER SqlCredential
 Use SqlCredential to connect to SqlInstance with SQL authentication. 
 If SqlCredential is not specified, Windows authentication will be used.
 
+.PARAMETER Silent
```

Comment:
This is part of 1.0 and it's not expected new contributors would know this. It wills soon be renamed to AllowExceptions but til then...